Antonio_Ramos_Documentation

```diff
@@ -0,0 +1,171 @@
+
+- going to open up data files referenced in K-function Local_3Stage.py in ArcMap to see if the data they contain properly corresponds to the data structures being described in the K-function Local_3Stage.py script 
+
+- cannot open ArcMap from windows start bar, continue getting notification that the application is not responding. Will attempt to open from the command prompt window  
+
+- finally able to open arcmap by starting it from the start bar and waiting for the program to repsond 
+
+- when adding the data that is referenced in the K-function Local_3Stage.py program to a blank map in ArcMap the block data (CA_Block_New.shp) appears well, however, the zip data (CA_ZCTA_New.shp) does not appear on the map and it seems that when I add it as a data layer nothing changes on the map, furthermore, the labs data (GoodLabs_New.shp) does not appears as points on the block data map but as one point far away from the block data map. 
+
+- evaluated attribute tables to ensure that the proper data column names in the table were being referenced by the K-function Local_3Stage.py program. All data table names are correctly referenced, however, there is also another attribute "geometry" which is being listed on lines 39-41 and I am not sure what that is doing, it does not appear in any of the attribute tables from the data files. 
+
+- when running line by line analysis of K-function Local_3Stage.py I again get the infinite simulation loops when running lines 104-137 together. Upon further review I did not get any errors running lines 104-119 together, but it does not get past 50 percent complete on the simluations, have to interupt program with "control C". Could be because I am running the lines independent of any data. Now, when I run lines 120-128 together I get "SIMULATION 1 Threw an error". Moreover, when I run line 122 in K-fucntion Local_3Stage.py, I get the following error...
+  "File "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\kfunction_3Stage.py", line 49, in countSimulation
+    fsFeatures['FS_PROBS'] = fsFeatures[[fieldFS]] / float(sum(fsFeatures[fieldFS]))
+
+TypeError: unsupported operand type(s) for +: 'int' and 'str'"
+
+-(now working in kfunction_3Stage.py script) when running lines 42-49 (so as to include the definition of "fsFeatures") no error occurs and script runs perfectly fine. Moreover, when running the entire function definied as "countSimulation" (lines 42-87) no error is retunred and function executes. Similarly when running the entire function defined as "pointSimulation" (which is referenced in line 122 of K-function Local_3Stage.py) no error is returned and function executes. So I am confused as to what is sending us into this infinite loop. I believe it might have to do with the data we are feeding into the programs but I am not sure why. As mentioned above the attribute names being referenced coincide between the program and the data. 
```

Comment:
I also think that looking at the data is the right place to start. If the block, zip and labs data are not all right on top of each other in space, I can easily imagine how that would cause problems for the simulation function.

Before we talk tomorrow, please explore the program(s) that create these "new" versions of the data files--you may be able to gain some insight into why Will needed to adjust the data, and anything that might have gone wrong in that process.

Reply:
Will do.